content/docs/command-reference/import-url.md

```diff
@@ -360,40 +362,33 @@ Running stage 'prepare' with command:
 
 ## Example: Transfer to remote storage
 
-When you have a large dataset in an external location, you may want to import it
-to your project without downloading it to the local file system (for using it
-later/elsewhere). The `--to-remote` option let you skip the download, while
-storing the imported data [remotely](/doc/command-reference/remote). Let's
-initialize a DVC project, and setup a remote:
+Normally, `dvc import-url` downloads the target data (to the <abbr>cache</abbr>)
+in order to link and track it locally. But what if there's not enough disk space
+for the download?
 
-```dvc
-$ mkdir example # workspace
-$ cd example
-$ git init
-$ dvc init
-$ mkdir /tmp/dvc-storage
-$ dvc remote add myremote /tmp/dvc-storage
-```
+One option is to setup an
+[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
+in a location that can handle the data. Another is to use the `--to-remote`
```

Comment:
I wonder if we need a full to-cache example in `import-url` (like in https://dvc.org/doc/command-reference/add#example-transfer-to-the-cache).

Reply:
> I wonder if we need a full to-cache example in import-url

`import-url` doesn't have the `to-cache` ability, but rather work on its own (like you just import an URL with it and it puts it to your cache, but not in a chunked way).

Reply:
Ah, good point. But then maybe we should explain about the chunking (transferring) a bit more... I'll review again ⌛

Reply:
But with `--to-cache` the chinked transfer does happen, right @isidentical ? Just double checking. Already updated in c3932123, PTAL.